content/sensu-go/6.8/release-notes.md

```diff
@@ -110,6 +111,34 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 6.8.2 release notes
+
+**October 6, 2022** &mdash; The latest release of Sensu Go, version 6.8.2, is now available for download.
+
+Sensu Go 6.8.2
+
+Read the [upgrade guide][1] to upgrade Sensu to version 6.8.2.
+
+**IMPROVEMENTS:**
+
+- The authentication module now logs successful (INFO) and unsuccessful (ERROR) login attempts.
+- Agent websocket connection logging now includes backend entity name.
+- When check output is truncated due to the [max_output_size][302] configuration, the events the check produces will include a `sensu.io/output_truncated_bytes` label.
+
+**FIXES:**
+
+- ([Commercial feature][295]) In the web UI, restored the silence function on the Entities page.
+- ([Commercial feature][295]) In the web UI, resource pages now automatically refresh after creating resources.
+- ([Commercial feature][295]) The web UI now displays pipeline definitions under the **RAW** tab on individual pipeline resource pages.
+- ([Commercial feature][295]) In the web UI, corrected the link to the entity reference in the Edit Entity modal.
+- ([Commercial feature][295]) In the web UI, errors displayed  when deleting and re-adding an asset from asset page have been addressed.
```

Comment:
I can't figure out what this item refers to (`In the web UI, errors displayed  when deleting and re-adding an asset from asset page have been addressed.`). I see it in the changelog, but I can't find the corresponding issue or PR that makes sense.